src/apps/chifra/pkg/types/types_trace.go

```diff
@@ -66,15 +66,26 @@ func (s *SimpleTrace) Model(showHidden bool, format string, extraOptions map[str
 	// EXISTING_CODE
 	// EXISTING_CODE
 
-	model := map[string]interface{}{
```

Comment:
The auto code generation wants to assign the first version and then overwrite it, but that's easy to fix. Nothing to do here, but you'll see how this works when I merge my PR for makeClass.